README.md

```diff
@@ -26,6 +26,9 @@ Ruby library that consumes SpaceX API
   - [Missions](#missions)
     - `SPACEX::Missions.info`
     - `SPACEX::Missions.info('mission_id')`
+  - [Launchpads](#launchpads)
+    - `SPACEX::Launchpad.retrieve_all`
```

Comment:
In favor for compatibility, I'd use `.info` to retrieve all, and `.info('site_id')`

Reply:
👍🏻